pkg/trino/models/settings.go

```diff
@@ -14,6 +14,7 @@ type TrinoDatasourceSettings struct {
 	URL                 *url.URL           `json:"-"`
 	Opts                httpclient.Options `json:"-"`
 	EnableImpersonation bool               `json:"enableImpersonation"`
+	JWTAccessToken      string             `json:"jwtAccessToken"`
```

Comment:
I'd name this just `accessToken` here any in other places.

Reply:
Done